src/app/static/src/tests/components/password/resetPassword.test.ts

```diff
@@ -93,7 +93,7 @@ describe("Reset password component", () => {
             "Por favor, solicite outra ligação aqui.",
             store);
         expect((wrapper.find("#request-new-link a").element as HTMLLinkElement).href)
-            .toEqual("http://localhost/password/forgot-password");
+            .toEqual("http://localhost:3000/password/forgot-password");
```

Comment:
I think this might just be because vitest is assuming a base url of http://localhost:3000 rather than http://localhost - I see you took out the url environment option from config, maybe that made a difference?

Reply:
so i had it in the config to begin with and it didnt make a difference, and then on further investigation i realised the environment options are per environment so you have to do
```javascript
environmentOptions: {
  jsdom: {
    url: "http://localhost"
  }
}
```
and then it works! so mystery solved